src/test/java/bit/javaoop/AbstractManagerTest.java

```diff
@@ -0,0 +1,37 @@
+package bit.javaoop;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AbstractManagerTest {
+
+    @Test
+    void canHire() {
```

Comment:
No to tak :).
Konwencja nazewnictwa z should - powinno być coś typu `shouldCanHire`
Brak rozdziału na kilka testów, testujących różne przypadki - np. `shouldCanHire` i `shouldCanNotHire`
Brak rozdziału na kilka KLAS - `FixedBudgetManagerTest` oraz `FixedEmployeesManager`. Testujemy przypadki szczególne, a nie ogólne. (ogólne też można wtedy by nam powstała trzecia klasa `AbstractManagerTest`, ale nie ma za bardzo czego tam testować :P)
Widzę zarys konwencji blokowej //given //when //then, lecz brak komentarzy ją zaznaczających.

Przykład jak mógłby wyglądać test w Twoim przypadku:
![image](https://user-images.githubusercontent.com/26125971/38262103-916d4470-376c-11e8-9c2a-fed1842cd822.png)